fix-appdata.patch

```diff
@@ -0,0 +1,39 @@
+From 388f5ef9301f41030f73bfab3602c3bdc4df8468 Mon Sep 17 00:00:00 2001
```

Comment:
This can be removed, I'll merge the pr upstream.

Reply:
https://github.com/dusansimic/dynamic-wallpaper/pull/86

Reply:
I've merged that pr just now. If that's everything related to these patches (looks like it is) I'll create a new patch release and we can move forward with this pr.